docker-compose.yml

```diff
@@ -28,11 +28,11 @@ services:
       - POSTGRES_PORT=${BACKUP_DB_PORT}
       - SCHEDULE=${BACKUP_SCHEDULE}
   bot:
-    image: ghcr.io/pumpkin-py/pumpkin-py:latest
+    image: ghcr.io/strawberry-py/strawberry-py:latest
     container_name: ${INSTANCE_NAME}-bot
     env_file: .env
     volumes:
-      - ./:/pumpkin-py/:z
+      - ./:/strawberry-py/:z
```

Comment:
@j-fiala Just a sanity check, but do you think this might break something?

Reply:
If the naming stays consistent in folder structure, then It should not make any problems. LGTM